File: Python/tests/test_choiceRT_ddm_single.py
import pytest

from hbayesdm.models import choiceRT_ddm_single


def test_choiceRT_ddm_single():
    _ = choiceRT_ddm_single(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_choiceRT_ddm_single.R
context("Test choiceRT_ddm_single")
library(hBayesDM)

test_that("Test choiceRT_ddm_single", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(choiceRT_ddm_single(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
